### PR TITLE
Create Quicksilver

### PR DESCRIPTION
**Pull request by Mooshua**
A small little function that cleans up property assignment after creating calls with basic built-in error handling. This should help with most create functions within the main scripts and shave a few minutes of work. :)

**Here's how it works:**
a) require module 
b) module returns function, use it like one.
c) call function with instance (int) and table (tab) formatted like {property = value, property = value}
(tip: just format the table's property definitions without the ["propertyName"] glamour, just do {propertyName = value}

Enjoy! 

Edit: Feel free to decline this request and just put the function directly into the scripts, it might be easier that way.

### CODE REVIEW

File: Quicksilver.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
+--[[
+  (C) 2019 TheFluffyCat / Mooshua (Github / Teverse)
```
Comment (Neztore) [inline Quicksilver.lua]: Copyright you?

File: Quicksilver.lua [hunk 2]
```diff
@@ -2,10 +2,13 @@
   (C) 2019 TheFluffyCat / Mooshua (Github / Teverse)
   
   Quicksilver.lua
-  
+  "Polish it quicker"
```
Comment (Neztore) [inline Quicksilver.lua]: Why? Is this quote really needed?

Comment (Mooshua) [reply Quicksilver.lua]: Nope. I'll remove that.

File: Quicksilver.lua (hunk 1 above)
Comment (Mooshua) [reply Quicksilver.lua]: Sure, I'll move it to teverse. Sorry.

**Comment by Mooshua** [comment]
Changes made. Teverse is now the copyright holder while giving attribution to me. Is this what you were looking for? (Horrible onboarding removed)

**Comment by Mooshua** [comment]
Found out about engine.construct. Quicksilver is now useless. :/

**Pull request closed without merging**